scripts/mod2_pcoa.py

```diff
@@ -37,6 +50,12 @@ def body_site(coords, mapping_file):
                      index_col='#SampleID')
     mf = mf.loc[o.site_ids]
 
+    if samples is None:
+        samples = mf.index
+    else:
+        samples = set(samples.split(',')).intersection(set(o.site_ids))
```

Comment:
oh, i see now. sorry. this is to support task farming it as its embarassingly parallel